Package.resolved

```diff
@@ -6,71 +6,62 @@
         "repositoryURL": "https://github.com/Carthage/Commandant.git",
         "state": {
           "branch": null,
-          "revision": "07cad52573bad19d95844035bf0b25acddf6b0f6",
-          "version": "0.15.0"
+          "revision": "ab68611013dec67413628ac87c1f29e8427bc8e4",
+          "version": "0.17.0"
```

Comment:
for some reason, SPM is resolving `Commandant` to 0.17.0, not 0.16.0 as determined on `Package.swift` 🤷‍♂️